app/src/components/inputs/inputDropdown/inputDropdown.jsx

```diff
@@ -30,74 +30,97 @@ const cx = classNames.bind(styles);
 export class InputDropdown extends Component {
   static propTypes = {
     displayedValue: PropTypes.string,
+    value: PropTypes.oneOfType([PropTypes.string, PropTypes.array]),
     options: PropTypes.array,
     multiple: PropTypes.bool,
     selectAll: PropTypes.bool,
     disabled: PropTypes.bool,
-    active: PropTypes.bool,
     onChange: PropTypes.func,
     onFocus: PropTypes.func,
     onBlur: PropTypes.func,
   };
 
   static defaultProps = {
     displayedValue: '',
+    value: '',
     options: [],
     multiple: false,
     selectAll: false,
     disabled: false,
-    active: false,
     onChange: () => {},
     onFocus: () => {},
     onBlur: () => {},
   };
+  state = {
+    opened: false,
+  };
   componentDidMount() {
     document.addEventListener('click', this.handleClickOutside);
   }
   componentWillUnmount() {
     document.removeEventListener('click', this.handleClickOutside);
   }
   onClickSelectBlock = (e) => {
-    e.stopPropagation();
-    this.props.active
-      ? (() => this.props.multiple && this.props.onBlur())()
-      : this.props.onFocus();
+    if (!this.props.disabled) {
+      this.setState({ opened: !this.state.opened });
+      e.stopPropagation();
+      this.state.opened
+        ? this.props.onBlur()
+        : this.props.onFocus();
+    }
   };
   handleClickOutside = (e) => {
     if (this.node.contains(e.target) && this.props.multiple) {
       return;
     }
     this.props.onBlur();
   };
+
+  handleChange = (value) => {
+    if (this.props.multiple) {
+      const idx = this.props.value.indexOf(value);
+      if (this.props.value.indexOf(value) > -1) {
```

Comment:
Duplicate call to `indexOf`, use the constant you have defiled on the previous line instead